Sources/Logging/Logging.swift

```diff
@@ -17,27 +17,57 @@
     import Glibc
 #endif
 
-/// This is the protocol a custom logger implements.
+/// A `LogHandler` is an implementation of a logging backend. You only need to worry about `LogHandler`s if you wish
+/// to implement your own logging backend. If you just want to use `swift-log`, please refer to the documentation of
+/// `Logger`.
 public protocol LogHandler {
-    // This is the custom logger implementation's log function. A user would not invoke this but rather go through
-    // `Logger`'s `info`, `error`, or `warning` functions.
-    //
-    // An implementation does not need to check the log level because that has been done before by `Logger` itself.
-    func log(level: Logger.Level, message: Logger.Message, metadata: Logger.Metadata?, file: String, function: String, line: UInt)
-
-    // This adds metadata to a place the concrete logger considers appropriate. Some loggers
-    // might not support this feature at all.
+    /// `log` is called when a `LogHandler` must emit a log message. There is no need for the `LogHandler` to check
+    /// if the `level` is above or below the configured `logLevel`, `Logger` already performed this check and
```

Comment:
add "as" before `Logger` to read as:

> there is no need to check ..., as Logger already performed this check

so the "as" leads into the reason for why